### ISSUE TITLE
external documentation site is unavailable

### ISSUE THREAD

**Issue by dwenzel** [ramsey/uuid#304]
## Describe the bug
The external documentation of this project is unavailable.
The site doesn't load.

## To Reproduce...

Steps to reproduce the behavior (include code examples, if applicable):

* load https://uuid.ramsey.dev/en/latest/

## Expected behavior

Documentation is displayed.

**Comment by ramsey** [comment]
Do the docs load for you now?

The docs are hosted at https://readthedocs.org, so maybe they had a service interruption, but the docs appear to load now. Let me know if you are still unable to load them.

Thanks!

**Comment by dwenzel** [comment]
Hi @ramsey and thx for your quick reply.

Unfortunately I'm still not able to load the docs.
Those are the URLs I tried:
* https://uuid.ramsey.dev/en/latest/
* https://uuid.readthedocs.io/
* http://uuid.rtfd.io/

All are taken from the project website on readthedocs.org (https://readthedocs.org/projects/uuid/)
Other sites (e.g. https://ddev.readthedocs.io) do load. Therefore I did not assume an error on readthedocs.org

I did a `dig` on the domains :
```shell
$ dig  uuid.readthedocs.io

; <<>> DiG 9.10.6 <<>> uuid.readthedocs.io
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 49919
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;uuid.readthedocs.io.		IN	A

;; ANSWER SECTION:
uuid.readthedocs.io.	300	IN	A	104.208.221.96

;; Query time: 24 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Sun Mar 29 20:24:33 CEST 2020
;; MSG SIZE  rcvd: 64

Atto:joh dw$ dig readthedocs.io

; <<>> DiG 9.10.6 <<>> readthedocs.io
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 54538
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;readthedocs.io.			IN	A

;; ANSWER SECTION:
readthedocs.io.		231	IN	A	104.17.33.82
readthedocs.io.		231	IN	A	104.17.32.82

;; Query time: 20 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Sun Mar 29 20:26:56 CEST 2020
;; MSG SIZE  rcvd: 75

```
Both subdomains are listed with the same IP address.

**Comment by ramsey** [comment]
I have readthedocs configured to always redirect to uuid.ramsey.dev. What do you see when you dig that domain? I’m wondering if there’s a DNS routing issue.

**Comment by dwenzel** [comment]
> I’m wondering if there’s a DNS routing issue.

You might be right.

`dig` returns the IP address of the main domain `readthedocs.io`  `104.17.22.82` 

```shell
$ dig uuid.ramsey.dev

; <<>> DiG 9.10.6 <<>> uuid.ramsey.dev
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 54039
;; flags: qr rd ra; QUERY: 1, ANSWER: 3, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;uuid.ramsey.dev.		IN	A

;; ANSWER SECTION:
uuid.ramsey.dev.	22789	IN	CNAME	readthedocs.io.
readthedocs.io.		300	IN	A	104.17.33.82
readthedocs.io.		300	IN	A	104.17.32.82

;; Query time: 73 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Sun Mar 29 23:15:31 CEST 2020
;; MSG SIZE  rcvd: 104
```

whereas a working subdomain is resolved as `104.208.221.96`

```
$ dig ddev.readthedocs.io

; <<>> DiG 9.10.6 <<>> ddev.readthedocs.io
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 63580
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;ddev.readthedocs.io.		IN	A

;; ANSWER SECTION:
ddev.readthedocs.io.	300	IN	A	104.208.221.96

;; Query time: 23 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Sun Mar 29 23:20:18 CEST 2020
;; MSG SIZE  rcvd: 64
```

**Comment by ramsey** [comment]
Closing this issue. I hope you're able to load the docs now.
